lib/ts/framework/awsLambda/framework.ts

```diff
@@ -286,7 +286,7 @@ export const middleware = (handler?: Handler): Handler => {
             }
             return response.sendResponse({});
         } catch (err) {
-            supertokens.errorHandler(err, request, response);
+            await supertokens.errorHandler(err, request, response);
```

Comment:
If this throws an error, what will happen?

Reply:
if the middleware is inside a try catch block, the error will go to that catch block. Else the lambda will throw the error and the API gateway return a 500 response